druid/src/widget/checkbox.rs

```diff
@@ -145,7 +147,13 @@ impl Widget<bool> for Checkbox {
                 .line_cap(LineCap::Round)
                 .line_join(LineJoin::Round);
 
-            ctx.stroke_styled(path, &env.get(theme::LABEL_COLOR), 2., &style);
+            let brush = if ctx.is_disabled() {
+                env.get(theme::DISABLED_FOREGROUND_LIGHT)
+            } else {
+                env.get(theme::LABEL_COLOR)
+            };
+
+            ctx.stroke_styled(path, &brush, 2., &style);
```

Comment:
Unfortunately this makes the checkmark not be visible at all, for me?

also: repeating myself, but I think the most important thing is to draw the text dimmed.

Reply:
Im using `TEXT_COLOR` and `DISABLED_TEXT_COLOR` now, does this change anything?